client/css/layout.css

```diff
@@ -295,6 +381,14 @@ body.aspectTooGood #hideToolbarButton {
   background-image:url(/i/icons-white/info-circle-solid-svg.svg);
 }
 
+#muteButton {
+  background-image:url(/i/icons-white/mute-svg.svg);
+}
+
+#muteButton.toolbarButton.muted {
+  background-image:url(/i/icons-white/unmute-svg.svg);
+}
```

Comment:
I would rather use font awesome icons just to keep the current style. if you need any new icon I can work on it.

https://fontawesome.com/v5.15/icons/volume-up?style=solid
https://fontawesome.com/v5.15/icons/volume-mute?style=solid